modules/oadp-dpt-use-case-azure.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-data-protection-test.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-dpt-use-case-azure_{context}"]
+= Running a data protection test on an Azure cluster
```

Comment:
title is a bit misleading. DPT is run on Azure object storage rather than Azure cluster